wger/exercises/views/muscles.py

```diff
@@ -118,4 +122,5 @@ def get_context_data(self, **kwargs):
         context['title'] = _(u'Delete {0}?').format(self.object.name)
         context['form_action'] = reverse(
             'exercise:muscle:delete', kwargs={'pk': self.kwargs['pk']})
+        cache.clear()
```

Comment:
@Rhytah, Could you possibly try using the `delete` method other than the clear because the `clear` clears all existing cache possibly available than the actual or specific cache you want to clear here.
Try out this link, i hope it will be helpfully:- https://stackoverflow.com/questions/8784400/clearing-specific-cache-in-django

Reply:
@Emmanuel-Dominic Let me try it out and get back to you.

Reply:
@Emmanuel-Dominic , According to this documentation, https://docs.djangoproject.com/en/2.2/topics/cache/ The function you are suggesting, cache.delete(), is used to delete a specific cache item using its key. According to the story card she is working on, they required her to reset the cache when a muscle has been deleted. The method she used does this. Good work @Rhytah

Reply:
@Emmanuel-Dominic I did try it out but the `cache.delete()` was not resetting exercise cache as desired. Hence the choice to maintain `cache.clear()` after reading up on your suggestion and documentation after consultations with @Manorlds-Eaglespark and @reifred as well.